service/src/test/java/org/cbioportal/service/impl/GenericAssayCategoricalDataServiceImplTest.java

```diff
@@ -0,0 +1,191 @@
+package org.cbioportal.service.impl;
+
+import org.cbioportal.model.*;
+import org.cbioportal.model.meta.GenericAssayMeta;
+import org.cbioportal.persistence.MolecularDataRepository;
+import org.cbioportal.service.GeneService;
+import org.cbioportal.service.GenericAssayService;
+import org.cbioportal.service.MolecularProfileService;
+import org.cbioportal.service.SampleService;
+import org.cbioportal.service.exception.MolecularProfileNotFoundException;
+import org.cbioportal.service.util.ExpressionEnrichmentUtil;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.math.BigDecimal;
+import java.util.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GenericAssayCategoricalDataServiceImplTest extends BaseServiceImplTest{
+    @InjectMocks
+    private GenericAssayCategoricalDataServiceImpl genericAssayCategoricalDataServiceImpl;
+    @Mock
+    private SampleService sampleService;
+    @Mock
+    private MolecularProfileService molecularProfileService;
+    @Mock
+    private MolecularDataRepository molecularDataRepository;
+
+    @Spy
+    @InjectMocks
+    private ExpressionEnrichmentUtil expressionEnrichmentUtil;
```

Comment:
It seems this is not being used anywhere, I know this is probably coming from our example test, but since we found it, let's remove it.

Reply:
Okay. I will have a look at your comments. Thanks for your efforts! @dippindots